packages/react-sdk/src/arg-types/utils.ts

```diff
@@ -30,77 +29,72 @@ export const propsToArgTypes = (
       return result;
     }
 
-    const control = mapControlForType(prop);
-
-    result[propName] = { ...prop, ...control };
+    const argType = getArgType(prop);
+    if (argType != null) {
+      result[propName] = argType;
+    }
     return result;
-  }, {} as ArgTypes);
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  }, {} as Record<string, any>);
```

Comment:
can't you use our zod type here?

Reply:
Can't import from bin because of rootDir in tsconfig.json